config/example/ir.yaml

```diff
@@ -19,6 +19,45 @@ morph:
   validators: # List of hex-encoded 33-byte public keys of sidechain validators to vote for at application startup
     - 0283120f4c8c1fc1d792af5063d2def9da5fddc90bc1384de7fcfdda33c3860170
   switch_interval: 2m # interval b/w RPC switch attempts if the node is not connected to the highest priority node
+  consensus: # Local consensus launch mode. Activates only when 'endpoint.client' is unset.
+    block_interval: 1s # Time period (approximate) between two adjacent blocks. Must be positive
+    magic: 15405 # Network magic. Must be unsigned integer in range [1:4294967295]
+    traceable_chain_length: 2102400 # Length of the chain accessible to smart contracts. Must be in range [1:4294967295]
+    committee: # Initial committee
+      - 02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2 # Hex-encoded public key
+    seed_nodes: # Optional list of existing nodes to communicate with over Neo P2P protocol. By default, node runs as standalone
+      # Same format as 'p2p.listen'
+      - node2:20333
+      - node3:20333
+    hardforks: # Optional hardforks
+      name: 1730000 # Maps name to chain height. Heights must not be greater than 4294967295
+    storage: # Blockchain storage
+      type: boltdb # One of following storage types:
+        # boltdb (local BoltDB)
+        # leveldb (local LevelDB)
+        # inmemory (volatile storage, mostly suitable for testing)
+        #
+        # If 'path' exists, it should correspond to the selected variant.
+      path: ./db/morph.bolt # File system path to the storage. Ignored for 'inmemory' storage.
+    rpc: # RPC settings
+      listen: # Optional list of network addresses to listen Neo RPC on. By default, protocol is not served
+        # TCP addresses in 'host:port' format
+        - localhost:30333
+        - localhost:30334
+    p2p: # P2P settings
+      dial_timeout: 5s # Maximum duration a single peer dial may take. Must be positive
+      proto_tick_interval: 2s # Time period between protocol ticks with each connected peer. Must be positive
+      listen: # Optional list of network addresses to listen Neo P2P on. By default, protocol is not served
+        # TCP addresses in 'host:port' format
+        - localhost:20333
+        - localhost:20334
+      peers:
+        min: 1 # Minimum number of peers a node needs for normal operation. Must be in range [0:2147483647]
```

Comment:
It should probably be noted that if being too large, this setting will prevent node from starting the consensus process. In other words, if the node is unable to connect to at least `min` peers, then it won't start dbft and (if `StartWhenSynchronized` is on) the RPC.

Reply:
NeoGo doesn't note that.

> too large

Can you please give a specific (albeit approximate) value? 100, 1000, 10000...?

Reply:
We mention some "optimal" numbers in https://github.com/nspcc-dev/neo-go/blob/master/docs/consensus.md#start-nodes-manually, but it all depends on the network. I bet private installations may have CNs only on the network in many cases.

Reply:
> Can you please give a specific (albeit approximate) value? 100, 1000, 10000...?

We're misunderstanding each other. Let the `min` be set to `2`. Then the node won't start accepting blocks until `2` peers are connected. Sometimes it may be unobvious why the node doesn't start the consensus process (it just stales in waiting for more peers to be connected), and Alex has encountered this problem. I'll add a comment about it to neo-go doc.